src/components/base-form.njk

```diff
@@ -31,18 +32,47 @@
       "role": "complementary"
     }
   }) }}
-
   {% block backLink %}
-    {% if backLink %}
-      {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
-      <span id="back">{{ govukBackLink({
-          text: translate("govuk.backLink"),
-          href: backLink
-        }) }}</span>
-    {% endif %}
+    <div class="govuk-width-container">
```

Comment:
Heyy Sophia, so since the mobile changes there's been a slight update to the way we format the back button with v2 if you have a little look at the [base.njk of mobile](https://github.com/govuk-one-login/mobile-id-check-frontend/pull/852/commits/1a186d3e1a440f836f407a380768f20dabace32f#diff-486508c7144d7721c4cdaf0f183366f60a2cdfae08f782c65c8a17b68fa901bb), we've got the spacing baked into the component now meaning that at pod level we can remove some of the <div> containers :) 
<img width="503" alt="Screenshot 2024-02-22 at 17 39 23" src="https://github.com/govuk-one-login/ipv-cri-common-express/assets/38694448/9bebce75-0c07-4151-80f4-3a218f0e8147">

Also when we put the toggle on the base file, we just need to ensure the toggle comes first then the back button
![Uploading Screenshot 2024-02-22 at 17.39.23.png…]()
 afterwards so that it stacks like this on mobile

Reply:
Also this is new as of yesterday! so you didn't miss it, i just hadn't pushed the change till yesterday!

Reply:
Added the screenshots to pr description and removed all the extra div containers :)